manual/manual/cmds/comp.etex

```diff
@@ -100,6 +100,17 @@ phase can be run directly, as in:
 The produced file has the executable bit set, and it manages to launch
 the bytecode interpreter by itself.
 
+Upon successful completion of the type checking procedure, type information is
+used to create an abstract typed syntax tree (AST). It contains complete
+information about the location and the specific type of each token in the
+source file.
+The compiler can output this tree as compiled ".cmt"/".cmti" files for the
+implementation/signature of the compilation unit. This operation is activated
+by passing the key "–bin-annot" to the compiler (see the descriprion of
```

Comment:
If I am not mistaken there is an en dash in`­–bin-annot`,  a simple hyphen-minus would be enough `-bin-annot`.
Also `description`.